### PR TITLE
Meta description

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

To add the description meta-tag in the bundle landing page.

[**Trello Card**](https://trello.com/c/3q5vqYh3)

### Before

picture 440

### CODE REVIEW

File: conf/strings.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+copy.bundleLanding.description="Help us deliver the independent journalism the world needs. Support the Guardian by making a contribution or getting a subscription."
```
Comment (davidfurey) [inline conf/strings.conf]: You could probably drop the `copy.` prefix

File: app/wiring/CopyConfiguration.scala [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package wiring
+
+import config.StringsConfig
+
+trait CopyConfiguration {
+  val copyConfig = new StringsConfig()
```
Comment (davidfurey) [inline app/wiring/CopyConfiguration.scala]: Change references from copy to string?

Comment (svillafe) [reply app/wiring/CopyConfiguration.scala]: Ouch, I forgot to update those. 🤕

**Comment by svillafe** [comment]
Evaluate if we will use the `strings.conf` file in the future. Trello card [here](https://trello.com/c/yDjjieKU/1071-evaluate-if-we-need-the-stringsconf-file)

**Comment by svillafe** [comment]
The meta tag is appearing in `/uk`

**Comment by svillafe** [comment]
It is working on **PROD**. Below a screenshot from [google.com](https://google.com) 
picture 442

**Pull request merged**